internal/apply/apply.go

```diff
@@ -2,54 +2,77 @@ package apply
 
 import (
 	"fmt"
+	"path/filepath"
 
 	"log"
 	"os/exec"
 	"path"
 	"strings"
 
+	"github.com/commitdev/zero/internal/module"
 	"github.com/commitdev/zero/internal/util"
-	"github.com/commitdev/zero/pkg/util/flog"
 
+	"github.com/commitdev/zero/internal/config/globalconfig"
 	"github.com/commitdev/zero/internal/config/projectconfig"
 	"github.com/commitdev/zero/pkg/util/exit"
+	"github.com/commitdev/zero/pkg/util/flog"
 	"github.com/manifoldco/promptui"
 )
 
-// Apply will bootstrap the runtime environment for the project
-func Apply(dir string, applyConfigPath string, applyEnvironments []string) []string {
-	context := loadContext(dir, applyConfigPath, applyEnvironments)
+func Apply(rootDir string, configPath string, environments []string) {
+	if strings.Trim(configPath, " ") == "" {
+		exit.Fatal("config path cannot be empty!")
+	}
```

Comment:
Would it be better to return the error in this case and let the caller throw the error for consistency like in #130?

Reply:
Nvm, I just realized your comment was specific to if I wanted to test the error case so I guess that doesn't apply here since it's not a standard.